src/pudl/transform/eia860.py

```diff
@@ -1021,3 +1027,124 @@ def _core_eia860__boiler_stack_flue(
     )
 
     return bsf_assn
+
+
+@asset(io_manager_key="pudl_io_manager")
+def _core_eia860__cooling_equipment(
+    raw_eia860__cooling_equipment: pd.DataFrame,
+) -> pd.DataFrame:
+    """Transform the EIA 860 cooling equipment table.
+
+    - spot clean year values before converting to dates
+    - standardize water rate units to gallons per minute (2009-2013 used cubic
+      feet per second)
+    - convert kilodollars to normal dollars
+
+    Note that the "power_requirement_mw" field is erroneously reported as
+    "_kwh" in the raw data for 2009, 2010, and 2011, even though the values are
+    in MW. This is corroborated by the values for these plants matching up with
+    the stated MW values in later years. Additionally, the PDF form for those
+    years indicates that the value should be in MW, and KWh isn't even a power
+    measurement.
+
+    In 2009, we have two incorrectly entered ``cooling_type`` values of ``HR``,
+    for utility ID 14328, plant IDs 56532/56476, and cooling ID ACC1. This
+    corresponds to the Colusa and Gateway generating stations run by PG&E. In
+    all later years, these cooling facilities are marked as ``DC``, or "dry
+    cooling"; however, ``HR`` follows the coding pattern for hybrid systems. As
+    such we drop the ``HR`` code completely in ``pudl.metadata.codes``.
+    """
+    ce_df = raw_eia860__cooling_equipment
+
+    # Generic cleaning
+    ce_df = ce_df.pipe(pudl.helpers.fix_eia_na).pipe(pudl.helpers.add_fips_ids)
+
+    # Spot cleaning and date conversion
+    ce_df.loc[
+        ce_df["chlorine_equipment_operating_year"] == 971,
+        "chlorine_equipment_operating_year",
+    ] = 1971
+    ce_df.loc[ce_df["pond_operating_year"] == 0, "pond_operating_year"] = np.nan
+    ce_df.loc[ce_df["tower_operating_year"] == 974, "tower_operating_year"] = 1974
+    ce_df = (
+        ce_df.pipe(pudl.helpers.month_year_to_date)
+        .pipe(pudl.helpers.convert_to_date)
+        .rename(columns={"operating_date": "cooling_system_operating_date"})
+    )
+
+    # There's one row which has an NA cooling_id_eia, which we make "NA" to
```

Comment:
Would it be more consistent to convert this to "PLANT", or does the plant have more than one cooling system associated with it?